x-pack/plugins/cases/server/services/user_actions/builder_factory.test.ts

```diff
@@ -185,6 +195,170 @@ describe('UserActionBuilder', () => {
     `);
   });
 
+  it('builds an external reference attachment (savedObject) user action correctly', () => {
```

Comment:
I added some tests for external references attachments that I missed in my previous PR.